astropy/io/fits/scripts/fitsheader.py

```diff
@@ -451,6 +458,20 @@ def main(args=None):
     if args.table is None:
         args.table = 'ascii.fixed_width'
 
+    # TODO: This is imho a little nicer than the custom action approach requiring
+    #  the store keyword.
+    #  Approach should be unified to whatever method is chosen
```

Comment:
Yes agreed, could you do the same for keywords ? Thanks.

Reply:
Done